recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/kernel-module-gles/kernel-module-gles_1.9.bbappend

```diff
@@ -0,0 +1,7 @@
+require inc/xt_shared_env.inc
+
+PVRKM_URL = "git://git@gitpct.epam.com/epmd-aepr/pvr_km_vgpu_img.git"
```

Comment:
I think, this is the same value as in `kernel-module-gles_1.9.bb`

Reply:
fixed now the base .bb, so these are different